src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,62 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { Dropdown as FabricDropdown, IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
+import { BaseComponentProps } from '../../util/BaseComponent/props';
+import './dropdown.css';
+
+export type DropdownOptionKey = string | number;
+
+export interface DropdownProps extends BaseComponentProps {
+  /**
+   * Items to be presented to the user.
+  **/
+  options: IDropdownOption[];
```

Comment:
Our interface here is leaking an implementation detail (the fact that we use Fabric under the hood). I would at least alias this type.

Also, you probably need to export it too, or your consumers won't be able to type their options properly.

Reply:
we're already surfacing a few Fabric interfaces where it makes sense, not to say we can't change that though. and here our component is named Dropdown so the name doesn't really give anything away. i'd agree that `DropdownOption` would be a better interface name than `IDropdownOption`

Reply:
Yeah, maybe `export type DropdownOption = IDropdownOption` would be enough.